ci-operator/jobs/openshift/installer/openshift-installer-presubmits.yaml

```diff
@@ -90,6 +90,64 @@ presubmits:
           - configMap:
               name: cluster-profile-aws
     trigger: ((?m)^/test( all| e2e-aws),?(\s+|$))
+  - agent: kubernetes
```

Comment:
@smarterclayton I'm not sure about this section, can you review this closely, thanks  
not sure about `TEST_FOCUS` and configmap-shared with other cluster-launch-installer-* tests?